.travis.yml

```diff
@@ -14,16 +14,25 @@ addons:
       - expect-dev # provides unbuffer utility
 
 stages:
+  - linting
   - test
 
 jobs:
   include:
+    - stage: linting
+      name: "pre-commit"
+      before_install:
+      install: pip install pre-commit
+      script: pre-commit run --all --show-diff-on-failure --verbose --color always
+      after_success:
     - stage: test
       env:
-        - TESTS=1 ODOO_REPO="odoo/odoo" MAKEPOT="1"
+        - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
     - stage: test
       env:
-        - TESTS=1 ODOO_REPO="OCA/OCB"
+        - TESTS="1" ODOO_REPO="OCA/OCB"
+          EXCLUDE="fetchmail_incoming_log,fetchmail_notify_error_to_sender_test"
+
 env:
   global:
     - VERSION="14.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
```

Comment:
IMO isn't a good idea have these changes to prevent errors in OCA/OCB, it's necessary to make all changes to pass all tests.

Reply:
yes, it's a good idea so. I add the cherry pic of your commit

Reply:
ok thanks, I have already remembered the reason for this change.